### ISSUE TITLE
trouble adding twitter as alternative auth

### ISSUE THREAD

**Issue by WardCunningham** [fedwiki/wiki-security-passportjs#6]
I have successfully configured twitter as an auth provider for bay.wiki.org and have even created and claimed a new site, test.bay.wiki.org. 

However, when I login with passport-persona and try to add twitter to an existing site I run through the twitter dialog but see no effect. Subsequent logins still offer only persona.

Here is the report from the server log.

```
GET /auth/addAuthDialog 200 1423 - 13.166 ms
GET /security/dialog.css 304 - - 1.329 ms
GET /favicon.png 304 - - 1.358 ms
GET /auth/twitter 302 0 - 147.235 ms
GET /auth/twitter/callback?oauth_token=t3-RGgAAAAAAyC2BAAABWIRelto&oauth_verifier=8wEK9W2MVTgxhvDnmi0q1ZQ7V0bQkvWs 302 74 - 890.615 ms
GET /auth/loginDone 200 1650 - 19.778 ms
GET /security/dialog.css 304 - - 2.579 ms
GET /security/winchan.js 304 - - 1.605 ms
rejecting - not authorized /auth/addAltAuth
GET /auth/addAltAuth 403 9 - 5.079 ms
GET / 302 43 - 2.368 ms
```

This is how it shows up in the browser.

![screen shot 2016-11-20 at 5 00 07 pm](https://cloud.githubusercontent.com/assets/12127/20467759/f077daf6-af42-11e6-8d0e-5f0a886ec9d5.png)

Here are the session cookies on the relevant request.

![screen shot 2016-11-20 at 5 06 19 pm](https://cloud.githubusercontent.com/assets/12127/20467828/d25212de-af43-11e6-8e6d-0ebc5c7dc230.png)

I will continue reading code and will comment here if I find something. But if I don't report success you can be sure I will be looking for suggestions.

**Comment by WardCunningham** [comment]
I should mention that the owner.json file is unchanged by this.
```
cat /root/.wiki/chifoo.bay.wiki.org/status/owner.json 
{"name":"Ward","persona":{"email":"ward@c2.com"}}
```

**Comment by paul90** [comment]
The changes in #7 should fix these problems

**Comment by paul90** [comment]
fix published as + wiki-security-passportjs@0.1.1
